src/app/query/stacks/balance/crypto-asset-balances.utils.ts

```diff
@@ -12,23 +12,20 @@ import type {
 import { createMoney } from '@shared/models/money.model';
 
 import { isTransferableStacksFungibleTokenAsset } from '@app/common/crypto-assets/stacks-crypto-asset.utils';
-import { formatContractId } from '@app/common/utils';
 
 export function createStacksCryptoCurrencyAssetTypeWrapper(
-  balance: BigNumber,
-  subBalance: BigNumber
+  balance: BigNumber
 ): StacksCryptoCurrencyAssetBalance {
   return {
     blockchain: 'stacks',
     balance: createMoney(balance, 'STX'),
+    type: 'crypto-currency',
```

Comment:
If relocating to migrate the discriminating union props to top, maybe move one more up to match others changed?